test/load/allocation/grpc/README.md

```diff
@@ -0,0 +1,107 @@
+# Load Test for gRPC allocation service
+
+This load tests aims to validate performance of the gRPC allocation service.
+
+## Kubernetes Cluster Setup
+
+For the load test you can follow the regular Kubernetes and Agones setup. In order to test the the allocation performance in isolation, we let the Agones to actuate all
+the game servers beefor starting the test.
+Here are the few important things:
+- Install Kubernetes in a region not zone to avoid being impacted by the rollouts
+- Consider using multiple cpu VMs for the agones-system node pool. We used n1-4 type VMs 
+- In the default-node pool (where the Game Server pods are created), make sure there are enough nodes available so Agones can actuate the game servers. We set the clustre count to 25 pre zonee (total of 75 per region)
+
+## Configuring the Allocator Service
+
+The allocator service uses gRPC. In order to be able to call the service, TLS and optionally mTLS has to be setup.
+For more information visit [agones-allocator]({{< relref "https://agones.dev/site/docs/advanced/allocator-service/">}}).
+
+## Fleet Setting
+
+We used the following fleet configuration during out testing. It creates 4000 simple-udp game servers. 
+Also uaing the automaticShutdownDelayMin parameter to 10, simple-udp game servers shutdown after 10 minutes.
+This helps to easily re-run the test without requiring to delete the game servers. 
+
+```yaml
```

Comment:
This looks like an inlined version of fleet.yaml (in the same PR). I worry they will get out of sync (DRY principle). 

Maybe just stick a link in here to the fleet.yaml file (it would be awesome if github would render a linked file inline like it does for images but I couldn't find a way to do that).

Reply:
Makes sense. I referenced the one from the master. I mentioned and highlight the changes. PTAL